ocs_ci/deployment/vmware.py

```diff
@@ -340,9 +340,13 @@ def deploy_prereq(self):
             self.kubeconfig = os.path.join(self.cluster_path, config.RUN.get('kubeconfig_location'))
 
             # git clone repo from openshift installer
+            # installer ( https://github.com/openshift/installer ) master and
+            # other branches (greater than release-4.3) structure has been
+            # changed. Use appropriate branch when ocs-ci is ready
+            # with the changes.
             clone_repo(
                 constants.VSPHERE_INSTALLER_REPO, self.upi_repo_path,
-                f'release-{get_ocp_version()}'
+                constants.VSPHERE_INSTALLER_BRANCH
```

Comment:
Vijay I would still stick with what we had: f'release-{get_ocp_version()}' so for 4.2 it will use 4.2 branch and for 4.3 - 4.3 branch only for 4.4 and higher we will use 4.3 branch as WA for now.

Based on conversation we had yesterday we will most likely will have to produce regular z streams for previous versions like 4.2, 4.3  (n-2, n-1) So would be good to keep content for specific version even it's working for 4.2 with 4.3.

WDYT?

Reply:
Chnaged the logic as per above suggestion. 
versions for ocs-ci that matches with ocs versions ?

Reply:
cloned the branch 4.3 for versions greater than 4.3
```
09:04:43 - MainThread - ocs_ci.utility.utils - INFO - Checking out repository to specific branch: release-4.3
09:04:43 - MainThread - ocs_ci.utility.utils - INFO - Executing command: git checkout release-4.3
09:04:46 - MainThread - ocs_ci.utility.utils - WARNING - Command stderr: Switched to a new branch 'release-4.3'

09:04:46 - MainThread - ocs_ci.utility.utils - INFO - Reset branch: release-4.3 with latest changes
09:04:46 - MainThread - ocs_ci.utility.utils - INFO - Executing command: git reset --hard origin/release-4.3
09:04:47 - Thread-1 - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_7.4)
09:04:48 - Thread-1 - paramiko.transport - INFO - Authentication (password) successful!
09:04:49 - MainThread - paramiko.transport.sftp - INFO - [chan 0] Opened sftp connection (server version 3)
09:04:49 - MainThread - ocs_ci.utility.utils - INFO - uploading /home/jenkins/current-cluster-dir/openshift-cluster-dir/bootstrap.ign to ocs4qe@dhcp35-153.lab.eng.blr.redhat.com:/var/www/html/ocs4qe/1589447029_bootstrap.ign
```